api/swagger-spec/v1beta3.json

```diff
@@ -12595,6 +12595,10 @@
       "type": "string",
       "description": "human readable message indicating details about why the pod is in this condition"
      },
+     "reason": {
+      "type": "string",
+      "description": "(brief-camelCase) reason indicating details about why the pod is in this condition"
```

Comment:
I have an issue with that. I see in kubelet that we use both formats:

[First Capital](https://github.com/GoogleCloudPlatform/kubernetes/blob/master/pkg/kubelet/kubelet.go#L1605)

[First Lower 1](https://github.com/GoogleCloudPlatform/kubernetes/blob/master/pkg/kubelet/kubelet.go#L1605)
[First Lower 2](https://github.com/GoogleCloudPlatform/kubernetes/blob/master/pkg/kubelet/kubelet.go#L1619)

Should I change my PR into `Reason` with the first letter capital and also change kubelet to reflect that?